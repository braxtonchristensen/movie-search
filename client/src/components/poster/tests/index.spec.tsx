import React from 'react';
import renderer from 'react-test-renderer';
import Poster, { PosterSize } from '../index';

describe('<Poster />', () => {
  describe('snapshots', () => {
    it('should match snapshot', () => {
      const rendered = renderer.create(
        <Poster
          size={ PosterSize.S }
          path="/gpxjoE0yvRwIhFEJgNArtKtaN7S.jpg"
          title="This will be dynamic and will be used for accessibility"
        />
      ).toJSON();

      expect(rendered).toMatchSnapshot();
    });
  });
});
