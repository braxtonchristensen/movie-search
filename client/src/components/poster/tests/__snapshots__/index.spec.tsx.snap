// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Poster /> snapshots should match snapshot 1`] = `
<img
  alt="This will be dynamic and will be used for accessibility"
  src="https://image.tmdb.org/t/p/w185/gpxjoE0yvRwIhFEJgNArtKtaN7S.jpg"
/>
`;
